mvlearn/utils/utils.py

```diff
@@ -94,30 +127,60 @@ def check_Xs_y(Xs, y, multiview=False, enforce_views=None):
         If provided, ensures this number of views in Xs. Otherwise not
         checked.
 
+    return_dimensions : boolean, (default=False)
+        If True, the function also returns the dimensions of the multiview
+        dataset. The dimensions are n_views, n_samples, n_features where
+        n_samples and n_views are respectively the number of views and the
+        number of samples, and n_features is a list of length n_views
+        containing the number of features of each view.
+
     Returns
     -------
     Xs_converted : object
         The converted and validated Xs (list of data arrays).
 
     y_converted : object
         The converted and validated y.
+
+    n_views : int, returned only if return_dimensions is True
+        The number of views in the dataset.
+
+    n_samples : int, returned only if return_dimensions is True
+        The number of samples in the dataset.
+
+    n_views : list, returned only if return_dimensions is True
+        List of lenght n_views containing the number of features for each
+        view.
     """
-    Xs_converted = check_Xs(Xs, multiview=multiview,
-                            enforce_views=enforce_views)
+    if return_dimensions:
+        Xs_converted, n_views, n_samples, n_features = check_Xs(
+            Xs,
+            multiview=multiview,
+            enforce_views=enforce_views,
+            return_dimensions=True,
+        )
+    else:
+        Xs_converted = check_Xs(
+            Xs, multiview=multiview, enforce_views=enforce_views
+        )
     _, y_converted = check_X_y(Xs_converted[0], y, allow_nd=False)
 
-    return Xs_converted, y_converted
+    if not return_dimensions:
+        return Xs_converted, y_converted
+    else:
+        return Xs_converted, y_converted, n_views, n_samples, n_features
```

Comment:
```suggestion
    if return_dimensions:
        return Xs_converted, y_converted, n_views, n_samples, n_features
    else:
        return Xs_converted, y_converted
```

For consistency and readability.